docker/README.md

```diff
@@ -32,6 +32,7 @@ The tag of docker image. The default value is `latest`.
 ### ES_VERSION
 
 The elasticsearch version this image supports. The default value is `es6`, available values are `es6` and `es7`.
+If the image version is 7.+,please set the value to es7.
```

Comment:
What does this mean?

Reply:
If the value is es6, the docker build reference is apache-skywalking-apm-bin.tar.gz, if the value of helm elastaticsearch.imageTag is 7.+, an error will be reported when deploying in k8s cluster. For details, see: [https://github.com/apache/skywalking/issues/4581]

Reply:
I mean, this sentence seems duplicated. The previous sentence says, there are two option values.

Reply:
This only gives the values of the two options, but does not indicate that the purpose of these two values is to select the files needed when build the image. This is just a kind reminder

Reply:
If you want to give the statement more clear, provide full message for both values. Maybe a table or list.

Reply:
The elasticsearch version this image supports. The default value is `es6`, available values are `es6` and `es7`.These two values means:
1. ‘es6’ is to build image through the apache-skywalking-apm-bin.tar.gz file;
2. ‘es7’ is to build image through apache-skywalking-apm-bin-es7.tar.gz file, the elasticsearch version used for storage is 7.+, use this value.

Reply:
What do you mean about above comments? I was suggesting you to update the document in this way.
And don't use 7+, ElasticSearch as 8.x releases already, which we never tested.